scrabble/README.md

```diff
@@ -1,9 +1,31 @@
-Scrabble
---------
+Word Game (Scrabble)
+--------------------
+![scrabble](https://www.word-grabber.com/3/m/2012/02/scrabble-rack-with-tiles.jpg)
+
+This is a game where players attempt to create words from a set of letter tiles.  Different letters have different scores
+
+* Calculate the score for a word. Use these scores for each letter:
```

Comment:
"defined as the sum of its letter point values"?

Reply:
Should we give an example?

```
GUARDIAN = 2 + 1 + 1 + 1 + 2 + 1 + 1 + 1 = 10
```